pb/config/config.proto

```diff
@@ -629,6 +631,20 @@ message Configuration {
   repeated DashboardGroup dashboard_groups = 3;
 }
 
+// A grouping of configuration options for the flakiness analysis tool.
+// Later configuration options could include the ability to choose different kinds of 
+// flakiness and choosing if and who to email a copy of the flakiness report.
+message HealthinessAnalysisOptions {
+  // Defaults to false; flakiness analysis is opt-in
+  bool should_analyze = 1;
```

Comment:
I think we can infer if this is enabled by checking if this message is nil or not, so should_analyze isn't needed?

Reply:
I was thinking that if I was going to add defaults to the default YAML then I would need a flag because the message would potentially be populated by the defaults. Although since that's not the case right now, should it be checking for nil now and later we add the should_analyze flag when we have defaulting functionality in?

Reply:
Ah actually, good point, I was not thinking about how to disable this is set in defaults but someone wants to opt-out. 

In that case, just a small nit: `enable` or similar for the field name?

(An alternative is to say, you have to set a positive days_of_analysis or we assume you didn't want to enable this, so people can opt out with days_of_analysis = 0. But I don't think that's as clear or necessary, unlike some other options where that makes more sense).